openupgradelib/openupgrade.py

```diff
@@ -2372,6 +2372,15 @@ def format_message(f):
         globaldict = {'uid': env.uid}
         if version_info[0] < 14:
             globaldict.update({'time': time})
+        if version_info[0] >= 13:
+            try:
+                from odoo.tools.safe_eval import datetime as safe_eval_datetime
+            except ImportError:
+                import datetime as safe_eval_datetime
+            globaldict.update({
+                'datetime': safe_eval_datetime,
+                'context_today': safe_eval_datetime.datetime.now,
+            })
```

Comment:
```suggestion
            try:
                from odoo.tools.safe_eval import datetime as safe_eval_datetime
                from odoo.tools.safe_eval import dateutil
            except ImportError:
                import datetime as safe_eval_datetime
                import dateutil
            globaldict.update({
                'datetime': safe_eval_datetime,
                'context_today': safe_eval_datetime.datetime.now,
                'relativedelta': dateutil.relativedelta.relativedelta
            })
``` 
@pedrobaeza I recently check it can have a case when relativedelta also disable too, dont you think?

Reply:
I added, thank you